### DISCUSSION TITLE
New DC water pump

### DISCUSSION THREAD

**Discussion by visualapproach** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#59] [Show and tell]
[new pump mod](https://youtube.com/shorts/7366tiENki8?feature=share)
My second water pump also broke so I ordered a cheap DC motor pump, 3D printed an adapter and switched the 12V AC transformer to a 12V DC. It's been running for a day now and it's MUCH quieter than the original. I wonder how long it will last.

**Comment by Bishbosh02** [comment]
Hi I upgraded my lazy spa to fibre glass shell and put all the electrics and heaters in a new box 3d printed the hose connectors 
![IMG_0135](https://user-images.githubusercontent.com/9953147/165744841-5ee1df7c-7ef5-4da0-9698-27a66ae6e83d.jpg)
So I'm interested in the pump you used and 3d printed parts...

**Comment by visualapproach** [comment]
First I 3d-printed this monsterosity to mate with the other half of the stock pump house. Driven by a 12V DC motor I had laying around. Think I got it from a butchered VHS player or something.  It was decently quiet for a week or so before going totally silent :(
![SPA pump part v19](https://user-images.githubusercontent.com/22987250/165830542-6fe64a77-4e4a-4c0a-bed1-d74d3544376f.png)
It was difficult to get it water tight, but it hold up for a while with rubber o-rings and silicone grease. Cannot recommend this method.
Then I bought a ready made little 12V pump from china:
![image](https://user-images.githubusercontent.com/22987250/165831312-177a7436-c7b8-4a3e-b35d-3c9e340fdd5b.png)
But it gave up after a few weeks. I think the brushes worn out or, as in the original pumps, the axle wore out. It doesn't take much before it jams.

So I got frustrated and bought two spare impellers with shafts and rubber end caps. First one is still functioning after 1000 hours of pumping.

The lid to my liner is punctured and there is a airleak in the liner so I was considering a hard tub. Can you tell me more about your project? It seems interesting!

**Reply by visualapproach** [reply]
I wasn't clear in last post. The impellers I bought was for the stock pump that came with the tub.

**Reply by tympietor** [reply]
Finding a replacement for the stock impellor pump is next on my list, even though I have now encapsulated the entire pump cube in an insulated box, suspended on (triple!!!) rubber mounts without touching the floor directly. Found out today that the droning vibrating noise mostly comes from the upper lid of the housing. So I unscrewed the top lid, added rubber foam tape around the lid where it mates with the lower shell of the housing, and loosely tightened the screws. Droning sound mostly gone now. All thanks to your project, I am able to leave the lid closed now, instead of having to open it for accessing the buttons :)
![20220423_103144](https://user-images.githubusercontent.com/99082697/166117687-83652e83-9c9f-4aa6-a5db-79ec43749cfd.jpg)
![20220424_142643](https://user-images.githubusercontent.com/99082697/166117656-145a677a-157a-47ca-b59f-7be9729a4951.jpg)
![20220423_180121](https://user-images.githubusercontent.com/99082697/166117667-1291367a-f20e-40f8-b3b2-7b2f49d78f1a.jpg)

About the pump; I have read on youtube that someone yanked out the original impellor pump, removed the transformer, and hooked up an aquarium pump with similar flow directly to the 230V coming to the original transformer. Seemed to work fine as well.

I am similarly looking in the area of aquarium pumps to find something suitable in the near future.

**Reply by visualapproach** [reply]
Wow it looks beautiful! Thanks for sharing!
And yeah, I think the aquarium business is where to look for alternatives to stock pump. It seems nobody sells water pumps that doesn't require it's own nuclear reactor or that can be used 24/7. I had an aquarium with a pump from eheim. Good quality but expensive. It lasted many years though, even if it's basically the same kind of impeller construction.

**Reply by Jay-Ti** [reply]
I mean the Pump has its own Power supply and gets switched by a relay on  (*insert country grid voltage here* has), Especially at this time around many people get rid of their old pool pumps. I got an Egg for cheap and plan on hooking up a 300W Pump (without the heating element because im heating the pool redneck style) But on a strong enough/ second Circuit/ Plug that should be fine, its not neatly and contained, but since Were all using an extra housing that also shouldnt be a problem.

**Comment by Bishbosh02** [comment]
Its not cheap process but picked up this Shell from ebay and all the pluming but come from https://www.hottubdiy.co.uk/ still need to do the sides and the hot tub cover is order but it 8 week delivery time
![s-l140](https://user-images.githubusercontent.com/9953147/166444932-921ef5f3-2db8-4bcf-88c0-bc764735adcb.jpg)
![IMG_0145](https://user-images.githubusercontent.com/9953147/166444941-6f12013a-a5d0-4ab1-9e35-f4e475490b04.jpg)
.

**Reply by visualapproach** [reply]
Ah, Thank you!

**Reply by Bishbosh02** [reply]
Some updated pics
![IMG_0248](https://user-images.githubusercontent.com/9953147/171998716-3952a0c4-d83d-45a9-baa9-ae5770acec7b.jpg)
![IMG_0249](https://user-images.githubusercontent.com/9953147/171998717-b105daa9-b6bc-45df-af4e-8cd52e2bd14c.jpg)
![IMG_0250](https://user-images.githubusercontent.com/9953147/171998722-22104a98-ac3a-44ac-9c55-4c37669b1072.jpg)

I also use an external Propane gas heater with 12V high pressure pump to heat the tub up and can heat it from 35Deg C to 38Dec C in 40 mins and the hottub holds 1500 litres .

The new custom lid keeps the heat in so it only looses 5 degrees over 8hrs.

**Reply by visualapproach** [reply]
Very nice! Ambitious project! Can you control the new pump and heater from the ESP? What kind of pump is it? Any links to where you bought it?

**Reply by Bishbosh02** [reply]
I dont use the esp to control the pump/light/airblower I do plan to change this but at the moment I use several shelly 1 pm with air switches and Home assistant 

Power Suppply
https://www.amazon.co.uk/gp/product/B06Y2BGRMZ/ref=ppx_yo_dt_b_asin_title_o05_s00?ie=UTF8&psc=1

Box for all the kit
https://www.amazon.co.uk/gp/product/B06ZYCDS3Q/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1

Pump
https://www.amazon.co.uk/gp/product/B09CPYCDSZ/ref=ppx_yo_dt_b_asin_title_o09_s00?ie=UTF8&th=1

Gass connector 
https://www.amazon.co.uk/gp/product/B07574WBWW/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

Hotwater Heater 
https://www.amazon.co.uk/gp/product/B08YQJQD34/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&th=1

Connectors 
https://www.amazon.co.uk/gp/product/B07L4983G2/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&th=1

Air Switches 
https://www.amazon.co.uk/gp/product/B09PH956TM/ref=ppx_yo_dt_b_asin_title_o03_s00?ie=UTF8&th=1
https://www.ebay.co.uk/itm/193518822774

**Reply by visualapproach** [reply]
Wow Thanks for all the links and info. Really interesting! I couldn't resist calculating the heat dissipation: 1500kg x 4.2kJ x 5 degree C / (8 h x 3600 s) = 1 kW. So continuous heating would need around 1 kW. That's 24 kWh per day. Don't know what that would be in BTU's ;)
I'm really inspired to make something similar myself now. If we could only have 48 hour days...
